pkg/gql/qontract_client.go

```diff
@@ -67,7 +67,7 @@ func newQontractConfig() *qontractConfig {
 }
 
 // NewQontractClient creates a new QontractClient
-func NewQontractClient(ctx context.Context) (*QontractClient, error) {
+func NewQontractClient(_ context.Context) (*QontractClient, error) {
```

Comment:
@janboll  can't we remove the refererences of ctx all together here as you did in   `getCredentialsFromVault` for uniformity accross code?

Reply:
It's not possible, since it's always set, see here:  https://github.com/Khan/genqlient/blob/v0.6.0/generate/operation.go.tmpl#L34C82-L34C82